app/assets/images/payment_icons/payflex.svg

```diff
@@ -1 +1 @@
.905 0 0 0 1.12-.12c.07-.07.14-.15.21-.22.34-.39.62-.81.84-1.26.19-.4.33-.83.42-1.26-.43.09-.86.23-1.26.42-.45.22-.87.5-1.26.84zM14.39 11.38c-.06-.2-.14-.39-.26-.54a1.24 1.24 0 0 0-.44-.38c-.18-.09-.39-.14-.63-.14-.25 0-.47.05-.65.15-.18.1-.32.23-.43.39-.11.16-.19.35-.24.55-.05.21-.08.42-.08.63 0 .22.03.44.08.65.05.21.14.39.25.55.12.16.26.29.45.39.18.1.4.15.66.15s.48-.05.65-.15c.17-.1.31-.23.42-.4.11-.17.18-.35.23-.57.05-.21.07-.43.07-.65 0-.22-.02-.43-.08-.63m-2.7-1.63v.62h.02c.12-.25.32-.44.58-.55.27-.12.56-.17.88-.17.35 0 .66.06.92.19s.48.31.66.53c.17.22.31.48.39.77.09.29.13.6.13.92 0 .32-.04.63-.13.92-.09.29-.22.54-.39.76-.17.22-.39.38-.65.51a2.187 2.187 0 0 1-1.3.15c-.14-.02-.28-.06-.41-.12a1.448 1.448 0 0 1-.69-.56h-.02v2.35h-.75V9.75h.76zM18.81 11.97c-.09.07-.22.12-.37.15a6 6 0 0 1-.48.08c-.17.02-.34.04-.51.07-.17.03-.32.07-.46.13s-.25.14-.33.25c-.09.11-.13.26-.13.44a.639.639 0 0 0 .27.52c.08.05.17.09.28.11.11.02.22.04.34.04.25 0 .46-.03.64-.1a1.204 1.204 0 0 0 .68-.59c.05-.12.08-.23.08-.34v-.76zm1.22 2.34c-.13.08-.31.11-.54.11-.19 0-.35-.05-.46-.16-.12-.11-.17-.29-.17-.54-.21.25-.45.43-.72.54-.27.11-.57.16-.89.16-.21 0-.4-.02-.59-.07a1.33 1.33 0 0 1-.48-.22c-.14-.1-.24-.23-.32-.39a1.32 1.32 0 0 1-.12-.59c0-.26.04-.47.13-.64.09-.17.2-.3.35-.4.14-.1.31-.18.5-.23.19-.05.38-.1.57-.13.21-.04.4-.07.59-.09.19-.02.35-.05.49-.09.14-.04.25-.09.34-.17.08-.07.12-.18.12-.32 0-.17-.03-.3-.09-.4a.59.59 0 0 0-.24-.23c-.1-.05-.21-.09-.33-.11-.12-.02-.24-.03-.36-.03-.32 0-.58.06-.8.18-.21.12-.33.35-.35.69h-.75c.01-.28.07-.52.18-.72.11-.19.25-.35.42-.47.18-.12.38-.21.61-.26.23-.05.47-.08.73-.08.21 0 .41.01.62.04.2.03.39.09.55.18.17.09.3.22.4.38.1.16.15.38.15.64v2.35c0 .18.01.31.03.39.02.08.09.12.21.12.06 0 .14-.01.23-.04v.6zM22.35 14.94c-.09.22-.18.41-.26.57-.09.15-.18.28-.28.38-.1.1-.22.17-.35.21-.13.04-.27.07-.44.07-.09 0-.18-.01-.27-.02-.09-.01-.17-.03-.26-.06v-.69c.06.03.14.05.23.08.09.02.16.03.22.03.15 0 .28-.04.39-.11.1-.07.18-.18.23-.31l.31-.77-1.81-4.55h.85l1.34 3.74h.02l1.28-3.74h.8l-2 5.17zM24.52 10.42v-.67h.77v-.68c0-.37.11-.65.32-.85.22-.19.53-.29.94-.29.07 0 .15.01.24.02s.17.03.24.05v.65c-.06-.02-.14-.04-.21-.05-.08-.01-.15-.01-.21-.01-.18 0-.32.04-.42.11-.1.07-.15.21-.15.41v.64h.88v.66h-.88v3.91h-.75v-3.91h-.77zM27.64 8.01h.75v6.32h-.75zM32.44 11.1c-.07-.16-.16-.3-.27-.41-.12-.12-.25-.21-.41-.27-.16-.07-.33-.1-.52-.1s-.37.03-.53.1c-.16.07-.29.16-.4.28-.11.12-.2.26-.27.41-.06.16-.1.32-.12.5h2.63c0-.18-.04-.35-.11-.51m.18 2.94c-.34.26-.77.39-1.29.39a2.3 2.3 0 0 1-.95-.18 1.88 1.88 0 0 1-.68-.5c-.18-.21-.32-.47-.41-.76-.09-.3-.14-.62-.16-.96 0-.35.05-.67.16-.96a2.176 2.176 0 0 1 1.13-1.25c.26-.12.55-.18.86-.18.41 0 .74.08 1.01.25.27.17.48.38.65.64.16.26.27.54.34.85.06.31.09.6.08.88h-3.43c-.01.2.02.39.07.57a1.248 1.248 0 0 0 .71.81c.18.08.4.12.65.12.32 0 .58-.07.78-.22.2-.15.34-.37.4-.67h.74c-.09.52-.31.91-.66 1.17"/><path class="st1" d="M33.42 9.75h.97l1.09 1.61 1.14-1.61h.91l-1.58 2.11 1.77 2.47h-.96l-1.28-1.89-1.27 1.89h-.91l1.71-2.41z"/></svg>
```

Comment:
- Please remove unused class `st1` and replace by direct attribute `fill`
- `width` and `height` attributes were missing
- Remove unused svg attributes `version="1.1" id="Layer_1"`

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" xml:space="preserve"><g style="opacity:.5"><path fill="#002751" d="M8.04 8.11zM6.88 15.81a.9.9 0 0 0 1.27 0c.03-.03.05-.06.08-.09-.02.03-.05.06-.08.09a.9.9 0 0 1-1.27 0zM.43 8.34zM8.15 9.47a.9.9 0 0 0 0-1.27c-.03-.03-.07-.06-.1-.09.04.03.07.06.1.09a.9.9 0 0 1 0 1.27zM.69 15.93c.25.16.56.18.83.07-.27.12-.58.09-.83-.07zM1.81 8.2a.9.9 0 0 0-1.27 0c-.04.05-.08.09-.11.14.03-.05.07-.09.11-.14a.9.9 0 0 1 1.27 0zM.76 14.33c.39-.34.81-.62 1.26-.84.4-.19.83-.33 1.26-.42.14-.7.14-1.43 0-2.13-.43-.09-.86-.23-1.26-.42-.45-.21-.87-.49-1.26-.84-.07-.07-.15-.14-.22-.21a.7.7 0 0 1-.13-.17c.04.06.08.12.13.17.07.07.13.14.19.21a3.589 3.589 0 0 1-.19 4.86c.07-.07.15-.14.22-.21zM8.22 15.72c.22-.28.24-.67.08-.97.17.31.14.69-.08.97zM.38 14.76a.898.898 0 0 0 .31 1.17.566.566 0 0 1-.15-.12.898.898 0 0 1-.16-1.05z"/><path fill="#002751" d="M6.67 10.52c-.4.19-.83.33-1.26.42-.14.7-.14 1.43 0 2.13.43.09.86.23 1.26.42.45.21.87.49 1.26.84.07.07.15.14.22.21a.8.8 0 0 1 .15.21c-.04-.07-.09-.14-.15-.21-.07-.07-.13-.14-.19-.21a3.589 3.589 0 0 1 .19-4.86c-.07.07-.15.14-.22.21-.39.34-.81.62-1.26.84zM2.86 9.68c.19.4.33.83.42 1.26.7.14 1.43.14 2.13 0 .09-.43.23-.86.42-1.26.21-.45.49-.87.84-1.26.07-.07.14-.14.21-.22-.07.07-.14.13-.21.19a3.589 3.589 0 0 1-4.65 0c-.07-.06-.14-.12-.21-.19.07.07.14.15.21.22.34.39.62.81.84 1.26zM5.83 14.33c-.19-.4-.33-.83-.42-1.26-.7-.14-1.43-.14-2.13 0-.09.43-.23.86-.42 1.26-.21.45-.49.87-.84 1.26-.07.08-.14.15-.21.22.07-.07.14-.13.21-.19a3.589 3.589 0 0 1 4.86.19c-.07-.07-.14-.15-.21-.22-.34-.39-.62-.81-.84-1.26z"/></g><path fill="#002751" d="M7.93 9.68c.07-.07.15-.14.22-.21a.9.9 0 0 0 0-1.27c-.03-.03-.07-.06-.1-.09a.894.894 0 0 0-.99-.04c-.06.03-.13.08-.18.13-.07.08-.14.15-.21.22-.34.39-.62.81-.84 1.26-.19.4-.33.83-.42 1.26.43-.09.86-.23 1.26-.42.45-.22.87-.5 1.26-.84zM2.02 8.42c-.07-.07-.14-.14-.21-.22a.9.9 0 0 0-1.27 0c-.04.05-.08.09-.11.14a.9.9 0 0 0 .11 1.13c.07.07.15.14.22.21.39.34.81.62 1.26.84.4.19.83.33 1.26.42-.09-.43-.23-.86-.42-1.26-.22-.45-.5-.87-.84-1.26zM8.15 14.54c-.07-.07-.15-.14-.22-.21-.39-.34-.81-.62-1.26-.84-.4-.19-.83-.33-1.26-.42.09.43.23.86.42 1.26.21.45.49.87.84 1.26.07.08.14.15.21.22a.9.9 0 0 0 1.27 0c.03-.03.05-.06.08-.09.22-.28.24-.67.08-.97-.05-.07-.1-.14-.16-.21zM.76 14.33c-.08.07-.15.14-.22.21a.898.898 0 0 0 .15 1.39.905.905 0 0 0 1.12-.12c.07-.07.14-.15.21-.22.34-.39.62-.81.84-1.26.19-.4.33-.83.42-1.26-.43.09-.86.23-1.26.42-.45.22-.87.5-1.26.84zM14.39 11.38c-.06-.2-.14-.39-.26-.54a1.24 1.24 0 0 0-.44-.38c-.18-.09-.39-.14-.63-.14-.25 0-.47.05-.65.15-.18.1-.32.23-.43.39-.11.16-.19.35-.24.55-.05.21-.08.42-.08.63 0 .22.03.44.08.65.05.21.14.39.25.55.12.16.26.29.45.39.18.1.4.15.66.15s.48-.05.65-.15c.17-.1.31-.23.42-.4.11-.17.18-.35.23-.57.05-.21.07-.43.07-.65 0-.22-.02-.43-.08-.63m-2.7-1.63v.62h.02c.12-.25.32-.44.58-.55.27-.12.56-.17.88-.17.35 0 .66.06.92.19s.48.31.66.53c.17.22.31.48.39.77.09.29.13.6.13.92 0 .32-.04.63-.13.92-.09.29-.22.54-.39.76-.17.22-.39.38-.65.51a2.187 2.187 0 0 1-1.3.15c-.14-.02-.28-.06-.41-.12a1.448 1.448 0 0 1-.69-.56h-.02v2.35h-.75V9.75h.76zM18.81 11.97c-.09.07-.22.12-.37.15a6 6 0 0 1-.48.08c-.17.02-.34.04-.51.07-.17.03-.32.07-.46.13s-.25.14-.33.25c-.09.11-.13.26-.13.44a.639.639 0 0 0 .27.52c.08.05.17.09.28.11.11.02.22.04.34.04.25 0 .46-.03.64-.1a1.204 1.204 0 0 0 .68-.59c.05-.12.08-.23.08-.34v-.76zm1.22 2.34c-.13.08-.31.11-.54.11-.19 0-.35-.05-.46-.16-.12-.11-.17-.29-.17-.54-.21.25-.45.43-.72.54-.27.11-.57.16-.89.16-.21 0-.4-.02-.59-.07a1.33 1.33 0 0 1-.48-.22c-.14-.1-.24-.23-.32-.39a1.32 1.32 0 0 1-.12-.59c0-.26.04-.47.13-.64.09-.17.2-.3.35-.4.14-.1.31-.18.5-.23.19-.05.38-.1.57-.13.21-.04.4-.07.59-.09.19-.02.35-.05.49-.09.14-.04.25-.09.34-.17.08-.07.12-.18.12-.32 0-.17-.03-.3-.09-.4a.59.59 0 0 0-.24-.23c-.1-.05-.21-.09-.33-.11-.12-.02-.24-.03-.36-.03-.32 0-.58.06-.8.18-.21.12-.33.35-.35.69h-.75c.01-.28.07-.52.18-.72.11-.19.25-.35.42-.47.18-.12.38-.21.61-.26.23-.05.47-.08.73-.08.21 0 .41.01.62.04.2.03.39.09.55.18.17.09.3.22.4.38.1.16.15.38.15.64v2.35c0 .18.01.31.03.39.02.08.09.12.21.12.06 0 .14-.01.23-.04v.6zM22.35 14.94c-.09.22-.18.41-.26.57-.09.15-.18.28-.28.38-.1.1-.22.17-.35.21-.13.04-.27.07-.44.07-.09 0-.18-.01-.27-.02-.09-.01-.17-.03-.26-.06v-.69c.06.03.14.05.23.08.09.02.16.03.22.03.15 0 .28-.04.39-.11.1-.07.18-.18.23-.31l.31-.77-1.81-4.55h.85l1.34 3.74h.02l1.28-3.74h.8l-2 5.17zM24.52 10.42v-.67h.77v-.68c0-.37.11-.65.32-.85.22-.19.53-.29.94-.29.07 0 .15.01.24.02s.17.03.24.05v.65c-.06-.02-.14-.04-.21-.05-.08-.01-.15-.01-.21-.01-.18 0-.32.04-.42.11-.1.07-.15.21-.15.41v.64h.88v.66h-.88v3.91h-.75v-3.91h-.77zM27.64 8.01h.75v6.32h-.75zM32.44 11.1c-.07-.16-.16-.3-.27-.41-.12-.12-.25-.21-.41-.27-.16-.07-.33-.1-.52-.1s-.37.03-.53.1c-.16.07-.29.16-.4.28-.11.12-.2.26-.27.41-.06.16-.1.32-.12.5h2.63c0-.18-.04-.35-.11-.51m.18 2.94c-.34.26-.77.39-1.29.39a2.3 2.3 0 0 1-.95-.18 1.88 1.88 0 0 1-.68-.5c-.18-.21-.32-.47-.41-.76-.09-.3-.14-.62-.16-.96 0-.35.05-.67.16-.96a2.176 2.176 0 0 1 1.13-1.25c.26-.12.55-.18.86-.18.41 0 .74.08 1.01.25.27.17.48.38.65.64.16.26.27.54.34.85.06.31.09.6.08.88h-3.43c-.01.2.02.39.07.57a1.248 1.248 0 0 0 .71.81c.18.08.4.12.65.12.32 0 .58-.07.78-.22.2-.15.34-.37.4-.67h.74c-.09.52-.31.91-.66 1.17"/><path fill="#002751" d="M33.42 9.75h.97l1.09 1.61 1.14-1.61h.91l-1.58 2.11 1.77 2.47h-.96l-1.28-1.89-1.27 1.89h-.91l1.71-2.41z"/></svg>
```